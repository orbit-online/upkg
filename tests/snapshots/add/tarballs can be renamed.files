├── [drwxr-xr-x]  .upkg
│   ├── [drwxr-xr-x]  .bin
│   │   └── [lrwxrwxrwx]  acme.bin -> ../.packages/acme-no-metadata.tar@8269ed897058ec43430168b2367ed8d588f2259ce6a373890dd0b1b96fd722c1/bin/acme.bin
│   ├── [drwxr-xr-x]  .packages
│   │   └── [drwxr-xr-x]  acme-no-metadata.tar@8269ed897058ec43430168b2367ed8d588f2259ce6a373890dd0b1b96fd722c1
│   │       └── [drwxr-xr-x]  bin
│   │           └── [-rwxr-xr-x]  acme.bin
│   └── [lrwxrwxrwx]  acme-2 -> .packages/acme-no-metadata.tar@8269ed897058ec43430168b2367ed8d588f2259ce6a373890dd0b1b96fd722c1
└── [-rw-r--r--]  upkg.json
