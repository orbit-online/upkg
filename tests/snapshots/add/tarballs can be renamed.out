upkg: Added '$BATS_RUN_TMPDIR/package-fixtures/acme-empty-v1.0.2-no-metadata.tar#name=acme-empty'
